judo-ui-react/src/main/resources/actor/src/components/table/ContextMenu.tsx.hbs

```diff
@@ -164,18 +154,19 @@ export const ContextMenu = forwardRef<ContextMenuApi, ContextMenuProps<any, any>
             {t('judo.component.Table.ContextMenu.copyText', { defaultValue: 'Copy text', columnName: selectedColumn?.headerName, field: field, value: selectedValue }) as string}
           </Typography>
         </MenuItem>
-        <MenuItem onClick={filterByCell}>
+        {/* Do not allow filtering for empty values */}
+        {(selectedValue !== null && selectedValue !== undefined) && <MenuItem onClick={filterByCell}>
           <MdiIcon path="equal" sx={ { mr: 1 } } />
           <Typography variant="inherit">
             {t('judo.component.Table.ContextMenu.filterBy', { defaultValue: 'Filter based on the selected value', columnName: selectedColumn?.headerName, field: field, value: selectedValue }) as string}
           </Typography>
-        </MenuItem>
-        <MenuItem onClick={excludeByCell}>
+        </MenuItem>}
+        {/*<MenuItem onClick={excludeByCell}>
           <MdiIcon path="not-equal-variant" sx={ { mr: 1 } } />
           <Typography variant="inherit">
             {t('judo.component.Table.ContextMenu.excludeBy', { defaultValue: 'Exclude based on the selected value', columnName: selectedColumn?.headerName, field: field, value: selectedValue }) as string}
           </Typography>
-        </MenuItem>
+        </MenuItem>*/}
```

Comment:
Commented code?

Reply:
Will be re-introduced once we figured out how to dance around not supported data-types.